mix.exs

```diff
@@ -190,8 +190,9 @@ defmodule Livebook.MixProject do
       logo: "static/images/logo.png",
       main: "readme",
       api_reference: false,
-      extra_section: "Docs",
+      extra_section: "Guides",
       extras: extras(),
+      filter_modules: fn mod, _ -> mod in [Livebook] end,
```

Comment:
Something like this?

```suggestion
      filter_modules: fn mod, _ -> mod in [Livebook] or String.starts_with?(inspect(mod), "Livebook.TestModules.") end,
```

Reply:
Ah, I removed a @moduledoc which I should have not. :)

Reply:
Ah sure that's filter is just for exdoc :D